test/metabase/api/app_test.clj

```diff
@@ -155,7 +155,18 @@
                                                                   :linkType "page",
                                                                   :targetId (:id detail-page)}}}
                                        {}]}
-                      list-page))))
+                      list-page))
+        (testing "Implicit actions are created"
+          (is (partial=
+                [{:slug "insert"}
+                 {:slug "update"}
+                 {:slug "delete"}]
+                (db/select ModelAction {:where [:= :model_action.card_id
+                                                {:select [:id]
+                                                 :from [Card]
+                                                 :where [:and
+                                                         [:= :collection_id (:collection_id app)]
+                                                         [:= :dataset true]]}]}))))))
```

Comment:
I expected an order-by clause here. Isn't it just a happenstance that this assertion is true?